src/kernel-install/50-depmod.install

```diff
@@ -33,6 +33,11 @@ case "$COMMAND" in
         exec depmod -a "$KERNEL_VERSION"
         ;;
     remove)
+        [ "$KERNEL_INSTALL_BOOT_ENTRY_TYPE" = "type2" ] || \
+        [ "$KERNEL_INSTALL_BOOT_ENTRY_TYPE" = "type1" ] && \
+        [ -d "/lib/modules/$KERNEL_VERSION/kernel" ] && \
```

Comment:
Hmm?? The condition looks spurious. At least not trivial, as it has `||` and `&&`...

Reply:
The idea is depmod shouldn't be removed when either type1 or type2 specified. Does this sound reasonable?